build.gradle

```diff
@@ -142,13 +142,13 @@ subprojects {
       'org.checkerframework.checker.nullness.NullnessChecker',
     ]
     if (runAllCheckers) {
-      // The experience shows that the below checkers are just rarely failing (if ever, in case of Regex), as compared to GuiEffect/Nullness.
+      // The experience shows that the below checkers are just rarely failing, as compared to GuiEffect/Nullness.
       // Hence, they're only applied in CI, or locally only if a dedicated variable is present in the env.
       checkers += [
         'org.checkerframework.checker.fenum.FenumChecker',
         'org.checkerframework.checker.index.IndexChecker',
+        'org.checkerframework.checker.interning.InterningChecker',
```

Comment:
Basically, check whether:
1. `equals` is unnecessarily used where `==` could be (less imporant)
2. `==` is used where `equals` should really be (more important)